src/ensembl/src/shared/components/instant-download/instant-download-protein/InstantDownloadProtein.tsx

```diff
@@ -43,14 +44,16 @@ const proteinOptionLabels: Record<keyof ProteinOptions, string> = {
 const InstantDownloadProtein = (props: InstantDownloadProteinProps) => {
   const [isProteinSeqSelected, setProteinSeqSelected] = useState(false);
   const [isCdsSeqSelected, setCdsSeqSelected] = useState(false);
+  const { genomeId, transcriptId } = props;
 
   const onProteinCheckboxChange = () =>
     setProteinSeqSelected(!isProteinSeqSelected);
   const onCdsCheckboxChange = () => setCdsSeqSelected(!isCdsSeqSelected);
 
-  const onSubmit = () => {
+  const onSubmit = async () => {
```

Comment:
1) why?
2) without the `await`, this functions is just immediately returning a Promise of undefined anyway

Reply:
This was a remnant of the previous attempt to get the download to work. It's removed now.

Reply:
✅